docs/transpile/customizing-transpilation.ipynb

```diff
@@ -0,0 +1,414 @@
+{
```

Comment:
<p>in this code example could you also include how to run the passmanger? and update the description above accordingly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/391/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Done.